### ISSUE TITLE
usb to hdmi adapter

### ISSUE THREAD

**Issue by iFeel-is-a-mouse** [teddywlq/smifb.kylin#1]
你好，如果是usb hdmi adapter可以用这个驱动不？有可能基于drm上面进行修改？估计难度有多大呢？

**Comment by teddywlq** [comment]
No. It can only support PCIe interface. Please download the USB driver from www.siliconmotion.com
